ios/UnityAds/UnityAdsVideo/UnityAdsVideoViewController.m

```diff
@@ -269,7 +269,12 @@ - (void)forceStopVideoPlayer {
 - (void)_createVideoPlayer {
   if (self.videoPlayer == nil) {
     UALOG_DEBUG(@"");
+      
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnonnull"
     self.videoPlayer = [[UnityAdsVideoPlayer alloc] initWithPlayerItem:nil];
+#pragma clang diagnostic pop
+      
```

Comment:
This change is a little sketchy; I'd like to know what you guys think @nikhilcn @Claytonious